### ISSUE TITLE
Extension of company with clang backend is copying the .dir file in wrong directory

### ISSUE THREAD

**Issue by frobino** [frobino/dot_emacs#5]

**Comment by frobino** [comment]
Fixed through the following: 

```
;; (interactive "sEnter project top folder: ")
 (interactive (list (read-directory-name "What directory? ")))
 (copy-file "~/.emacs.d/dot_dir_locals.el" prj_top_folder)
 (rename-file (concat prj_top_folder "dot_dir_locals.el")  (concat prj_top_folder ".dir.locals.el"))
```

wait for merge before closing

**Comment by frobino** [comment]
Fixed in 2bcadddc9a083447aabcf1725ffc8b87b4a2c9f8
